test-packages/test-services-odata-v4/test-service/TestEntityApi.ts

```diff
@@ -500,7 +504,8 @@ export class TestEntityApi<
         DURATION_PROPERTY: fieldBuilder.buildEdmTypeField(
           'DurationProperty',
           'Edm.Duration',
-          true
+          true,
+          0
```

Comment:
[q] Is there an example where nothing is set? Couldn't find it

Reply:
yes, for e.g, in the same file:
<img width="631" alt="Screenshot 2025-01-27 at 11 35 50" src="https://github.com/user-attachments/assets/9309bc46-26fc-4cc4-8e34-0a9f31e60574" />